easypost/services/report_service.py

```diff
@@ -0,0 +1,60 @@
+from typing import (
+    Any,
+    Dict,
+    List,
+    Optional,
+)
+
+from easypost.easypost_object import convert_to_easypost_object
+from easypost.models.report import Report
+from easypost.requestor import (
+    RequestMethod,
+    Requestor,
+)
+from easypost.services.base_service import BaseService
+
+
+class ReportService(BaseService):
+    def __init__(self, client):
+        self._client = client
+        self._model_class = Report.__name__
+
+    def create(self, **params) -> Report:
+        """Create a Report."""
+        url = f"{self._class_url(self._model_class)}/{params.pop('type')}"
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**question:** ‏ What happens if `type` is not included in the params? Should we do a guard-clause check first?

Reply:
Errors abound, I'll add a note to fix this when I overhaul error handling